core/src/test/scala/org/apache/spark/deploy/history/FsHistoryProviderSuite.scala

```diff
@@ -1070,7 +1137,7 @@ class FsHistoryProviderSuite extends SparkFunSuite with Matchers with Logging {
     clock.advance(TimeUnit.DAYS.toMillis(2))
     provider.checkForLogs()
     provider.cleanLogs()
-    assert(new File(testDir.toURI).listFiles().size === validLogCount)
+    assert(new File(testDir.toURI).listFiles().length === validLogCount)
```

Comment:
Actually, this is not for RocksDB.

Reply:
Sorry, Intellij automatically made code cleanup, and [12c0935](https://github.com/apache/spark/pull/36119/commits/12c09357b7f0d1bfc3b70a5f8c2f51d259a055e7) and [25e2699](https://github.com/apache/spark/pull/36119/commits/25e2699f64668f0bef6fe6c6c23f1097e5186262) revert the irrelevant changes